sequencer/api/availability.toml

```diff
@@ -0,0 +1,60 @@
+[route.getnamespaceproof]
+PATH = ["block/:height/namespace/:namespace"]
+":height" = "Integer"
+":namespace" = "Integer"
+DOC = "Get the namespace proof for a set of VM transations and the NMT root"
+
+[route.gettimestampwindow]
+PATH = [
+    "headers/window/:start/:end",
+    "headers/window/from/:height/:end",
+    "headers/window/from/hash/:hash/:end",
```

Comment:
What's the use case for this? It seems a bit strange to specify one edge of the interval with a hash and the other one with a timestamp.

Reply:
Use case for `from/hash` is the same as `from/:height`. It's if you've already fetched up to a certain block within the window, you can fetch the remaining blocks starting from there. I'm just trying to be consistent with the rest of the API, where whenever you specify a specific block, we accept either a block number or a hash